### PR TITLE
Meetup Fix #4 & Twitter Fix #3

### LINKED ISSUE #3: Include Twitter Feed on page
Use Twitter API to add latest tweets from Open Source Diversity to the website. 

Need to decide where this should go on page. Also include a follow button?

### LINKED ISSUE #4: Include latest meetup infomation on page
Use Meetup API to add latest Meetup data on webpage in events section.

### PR DESCRIPTION
**Pull request by sleepypioneer**
Fix #4 & Twitter Fix #3
This takes over from #8 
- [X] Add boxes for event information
- [X] Adds twitter API
- [x] Add Future Call information [Diversity & Inclusion calls](https://wiki.mozilla.org/Diversity_and_Inclusion_for_Communities_and_Contributors#Future_Calls)
- [X] Adds Meetup scraper & puts in initial data
- [ ] Design for Meetup Section
- [ ] Still needs styling to be added but hopefully it is clear where to do this in `meetupAPI.js` which is where I create the elements.
- [ ] Need to add a clearer link to the meetup main page.
- [ ] Each element does have a `href` to the individual meetup but it's touch zone is limited - this needs changing.
- [ ]  Also for some reason the Twitter button is ugly and not always working, I am not too sure why.

### CODE REVIEW

**Comment by sleepypioneer** [comment]
Screenshot from 2019-03-18 23-51-10

**Comment by sleepypioneer** [comment]
Hi @emmairwin I have opened this new PR for the events/ activities section as it had been so long since I worked on it last. I tried to put the D&I future calls in an iframe but I see on the wiki.mozilla.org website this option has been forbidden. Can you share perhaps where this information is fed in from? Perhaps I can access it from there also?

**Comment by sleepypioneer** [comment]
I've hard coding the information from D& I in for now, perhaps we can style it a bit nicer while I work out how we actually get that data? Going to leave it for now but happy to also clean it up a little myself. I also put a script in for getting the data with a ajax call but this currently has the same issue as the iframe.

Screenshot from 2019-03-19 00-31-11

File: index.html [hunk 1]
```diff
@@ -406,6 +441,22 @@ <h3>Programming Diversity</h3>
...
+                fjs.parentNode.insertBefore(js, fjs);
+				 t._e = [];
+                t.ready = function(f) {
+                    t._e.push(f);
+                };
+				 return t;
+            }(document, "script", "twitter-wjs"));
 		</script>
 	</body>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (jancborchardt) [inline index.html]: @sleepypioneer there’s the duplication of jQuery again. :) Do we need the full 3.3.1 or is the slim one in line 433 enough? If we need the full one, we can download it and update in the repository to not serve stuff from Google.

Comment (sleepypioneer) [reply index.html]: Sorry I commented on the old PR I need it or the ajax call was not working.

Comment (jancborchardt) [reply index.html]: Pushed some fixes, I updated the jQuery :)

Comment (jancborchardt) [reply index.html]: I rebased on master also so you need to `git pull --rebase` – sorry about the mess. :O

**Comment by jancborchardt** [comment]
Added some commits, looks like this now:
Events


In order to prevent having this very cool functionality lie around for some months again I’d say we merge it! :) Any further improvement can come in follow-up pull requests. What do you think @sleepypioneer?

**Comment by sleepypioneer** [comment]
Yeh I'd definitely like to get it looking a bit prettier sooner rather than later. but would merge this and open a new PR for it once we have a design :)

So I will go ahead and press green when I get a second review :+1: 

Great team effort @jancborchardt @earyzhe @colbye9 feeling satisfied :)

**Comment by jancborchardt** [comment]
Any other review @drashti4 @22nds @jonatoni or anyone? Otherwise let’s merge on Sunday to not let it lay around? :)

**Comment by jancborchardt** [comment]
Live now and it looks really nice! :)

**Pull request merged**